[Bug]: build failed  lua-language-server arm+i686

### Problem description

                                                   ^
3rd/bee.lua/bee/net/socket.cpp:545:77: error: cannot initialize a parameter of type 'socklen_t *' (aka 'int *') with an rvalue of type 'bee::net::socklen_t *' (aka 'unsigned int *')
        const int ok   = ::getsockopt(s, SOL_SOCKET, SO_ERROR, (char*)&err, &errl);
                                                                            ^~~~~
/home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/sys/socket.h:315:92: note: passing argument to parameter '__value_length' here
__socketcall int getsockopt(int __fd, int __level, int __option, void* __value, socklen_t* __value_length);
                                                                                           ^
5 errors generated.
[5/85] Compile C++ build/obj/source_bee/filewatch_linux.obj
[6/85] Compile C++ build/obj/source_bee/format.obj
ninja: build stopped: subcommand failed.
Error: Process completed with exit code 1.


[logs_31.zip](https://github.com/termux/termux-packages/files/11549532/logs_31.zip)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/packages/lua-language-server/bee.lua-bee-net-endpoint.patch
+++ b/packages/lua-language-server/bee.lua-bee-net-endpoint.patch
@@ -1,0 +1,13 @@
+--- a/3rd/bee.lua/bee/net/endpoint.h.orig
++++ b/3rd/bee.lua/bee/net/endpoint.h
+@@ -11,6 +11,10 @@
+ namespace bee::net {
+ #if defined(_WIN32)
+     using socklen_t = int;
++#elif defined(__ANDROID__) && !defined(__LP64__)
++    using socklen_t = std::int32_t;
++#elif defined(__ANDROID__) && defined(__LP64__)
++    using socklen_t = std::uint32_t;
+ #else
+     using socklen_t = unsigned int;
+ #endif

